hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/block/DeletedBlockLogImpl.java

```diff
@@ -285,16 +285,29 @@ public void addTransactions(Map<Long, List<Long>> containerBlocksMap)
     lock.lock();
     try {
       ArrayList<DeletedBlocksTransaction> txsToBeAdded = new ArrayList<>();
+      long currentBatchSizeBytes = 0;
+      final long maxBatchSizeBytes = 25 * 1024 * 1024; // 25MB
```

Comment:
@aryangupta1998 , thanks for working on this. Instead of use a fixed value,  we should get the current SCM raft message max length property value, and use that as a reference. 

You can refer to the DirectoryDeletingService for reference.

Reply:
Done